src/main/java/controller/ConsoleHandler.java

```diff
@@ -0,0 +1,89 @@
+package controller;
+
+import dao.BetDao;
+import dao.UserDao;
+import dao.UserDaoImpl;
+import java.util.List;
+import java.util.Scanner;
+import lib.Inject;
+import model.Bet;
+import model.User;
+
+public class ConsoleHandler {
+    private static final int DATA_QUANTITY = 2;
+    private static final int LOGIN_PASSWORD_MIN_LENGTH = 3;
+
+    @Inject
+    private UserDao userDao = new UserDaoImpl();
+    @Inject
+    private BetDao betDao;
+
+    public void handle() {
+        Scanner scanner = new Scanner(System.in);
+        User user = enterLoginPassword(scanner);
+        enterValueAndRisk(scanner, user);
+    }
+
+    private User enterLoginPassword(Scanner scanner) {
+        System.out.println("Введите логин и пароль:");
+        String creds = scanner.nextLine();
+        String[] userData = creds.split(" ");
+        if (userData.length != DATA_QUANTITY) {
+            handle();
```

Comment:
this is recursion
let's maybe refactor it

Reply:
done